main/accounts/index.js

```diff
@@ -372,6 +373,68 @@ class Accounts extends EventEmitter {
         currentAccount.update()
       }
     })
+    // If the account has any current requests, make sure fees are current
+    this.updatePendingFees()
+  }
+
+  setRecentFeeUpdate (currentAccount, id) {
+    // Set recently updated fee flag on tx and push update
+    currentAccount.requests[id].recentFeeUpdate = true
+    currentAccount.update()
+    
+    // Unset that flag after 3000ms
+    clearTimeout(this.timers[currentAccount.id + ':' + id])
+    this.timers[currentAccount.id + ':' + id] = setTimeout(() => {
```

Comment:
think we need to grab the current account again after the timeout or else it could potentially update a removed account